apps/revolt/config.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "../schema.json",
+  "name": "Revolt Chat",
+  "port": 8272,
+  "available": true,
+  "exposable": true,
+  "force_expose": true,
+  "generate_vapid_keys": true,
+  "supported_architectures": ["amd64"],
+  "id": "revolt",
+  "tipi_version": 1,
+  "version": "1.1.8/20230421-3",
+  "categories": ["media"],
+  "description": "Open source user-first chat platform.",
+  "short_desc": "Open source user-first chat platform.",
+  "author": "RevoltChat",
+  "source": "https://github.com/revoltchat",
+  "form_fields": [
+    {
+      "type": "random",
+      "label": "Revlot Minio Root Password",
+      "min": 32,
+      "env_variable": "REVOLT_MINIO_ROOT_PASSWORD"
+    },
+    {
+      "type": "random",
+      "label": "Revlot Minio Root Password",
+      "min": 32,
+      "env_variable": "REVOLT_MINIO_ROOT_PASSWORD"
+    },
+    {
+      "type": "random",
+      "label": "Revlot AWS Access Key",
+      "min": 16,
+      "max": 16,
+      "env_variable": "REVOLT_AWS_ACCESS_KEY_ID"
+    },
+    {
+      "type": "random",
+      "label": "Revlot AWS Secret Access Key",
+      "min": 32,
+      "max": 32,
+      "env_variable": "REVOLT_AWS_SECRET_ACCESS_KEY"
```

Comment:
Shouldn't this be a real AWS secret key? Instead of randomly generated one?

Reply:
We are deploying minio, so thats why its generated.